packages/next/telemetry/trace/trace.ts

```diff
@@ -83,3 +83,5 @@ export class Span {
 export const trace = (name: string, parentId?: SpanId, attrs?: Object) => {
   return new Span(name, parentId, attrs)
 }
+
+export const flushAllTraces = () => reporter.flushAll()
```

Comment:
Is this a no-op when there is no trace reporter?

Reply:
Yes 👍